client/src/components/layers/AnnotationLayer.vue

```diff
@@ -67,6 +67,32 @@ export default {
           this.$emit('annotation-right-click', e.data.record, e);
         }
       });
+    /* cancelOnMove = true will prevent clicks from registering if the user moves the cursor at all
+     this can make it hard to click on items
+     This will change the value based on being inside an annotation and amount of mouse movement */
+    this.polygonFeature.geoOn(geo.event.feature.mousemove, (e) => {
+      const calcVel = Math.abs(e.mouse.velocity.x) + Math.abs(e.mouse.velocity.y);
+      const interactorOpts = this.annotator.geoViewer.interactor().options();
+      const { cancelOnMove } = interactorOpts.click;
+      if (cancelOnMove && calcVel < 1) {
+        interactorOpts.click.cancelOnMove = false;
+      } else if (!cancelOnMove) {
+        interactorOpts.click.cancelOnMove = true;
+      }
+      // Only change the cancelOnMove if it is different
+      if (cancelOnMove !== interactorOpts.click.cancelOnMove) {
+        this.annotator.geoViewer.interactor().options(interactorOpts);
+      }
+    });
+    // If the user is not over any annotation we can set cancelOnMove=true
+    this.polygonFeature.geoOn(geo.event.feature.mouseoff, () => {
+      const interactorOpts = this.annotator.geoViewer.interactor().options();
+      const { cancelOnMove } = interactorOpts.click;
+      interactorOpts.click.cancelOnMove = true;
+      if (cancelOnMove !== interactorOpts.click.cancelOnMove) {
+        this.annotator.geoViewer.interactor().options(interactorOpts);
+      }
+    });
```

Comment:
I'm pretty hesitant to subscribe to mousemove events.  If GeoJS is discarding click events because of small amounts of mouse movement, I would consider that a GeoJS bug.

I'd like to add @manthey to this thread and see if this is really the best way to accomplish what we need.

Also, do we need to add code to unsubscribe too?  Will we end up with tons of detached mousemove listeners.?

Reply:
They have the `click.duration` time but then that means your clicks have to be deliberate clicks within a specified timeframe.

I'll double check in GeoJS and see if the destroy and `removeChild` removes all bound event handlers or not for any created features.

Reply:
There are two GeoJS settings regarding clicks: `cancelOnMove` and `duration`.  The default is `cancelOnMove=true` and `duration=0` (no limit).  This means that any movement isn't a click -- it is a pan.  This is a design intent for making it easy to pan a map in small increments.  If you set `cancelOnMove` to `false` and a `duration` in milliseconds, then clicks have to be shorter than the specified duration but can have movement, and panning works if it is slower than the specified duration.  I could see adding an additional movement threshold which would break small pans but not have the duration limit.

Reply:
All handlers bounds with geoOn are unbound when the associated element is destroyed.